evap/grades/views.py

```diff
@@ -117,29 +117,19 @@ def upload_grades(request, semester_id, course_id):
 
 
 @grade_publisher_required
-def toggle_no_grades(request, semester_id, course_id):
-    semester = get_object_or_404(Semester, id=semester_id)
+def toggle_no_grades(request):
+    course_id = request.POST.get("course_id")
     course = get_object_or_404(Course, id=course_id)
```

Comment:
you mean instead of just passing the `data_id`? what's the advantage? i never really liked the current structure where you have to hand over unnecessary variables (like the semester_id, when you're actually working on a course which is already unique by its own id).